NetCom2.iml

```diff
@@ -13,6 +13,7 @@
     <orderEntry type="library" scope="TEST" name="Maven: org.hamcrest:hamcrest-all:1.3" level="project" />
     <orderEntry type="library" scope="TEST" name="Maven: org.hamcrest:hamcrest-all:1.3" level="project" />
```

Comment:
What's all of this?

Reply:
I don't know, why the heck this is posted.. This file is contained within .gitignore...

Reply:
Must have been added before the .gitignore was there. Can we remove this file still?

Reply:
Sure, we should be able to. TBD before release 1.0